### DISCUSSION TITLE
fatal error after installing docker on Synology docker manager

### DISCUSSION THREAD

**Discussion by alhelo** [kasmtech/KasmVNC#217] [Q&A]
Dear team,
when I install ay image from Kasm on my synology docker manager,
it will be installed but in logs you can see alot of error and the system is not accessable.
I install it over scropt:
docker run -d \

docker run -d \
  --name=jummyDT\
  --security-opt seccomp=unconfined `#optional` \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=Europe/Berlin\
  -p 5004:6901 \
  -v /volume1/docker/ksm:/config \
  --restart unless-stopped \
   kasmweb/ubuntu-jammy-desktop:1.14.0

**after installation, when checking the folder ksm, it is empty.
and when try to access over UR:Port, I get error 402 empty response**

under logs I see the follwoing entries:

![image](https://github.com/kasmtech/KasmVNC/assets/5740797/17f565ac-2be9-4ac6-b23a-d4a43e5f06f3)

**Comment by alhelo** [comment]
container seems up and running

![image](https://github.com/kasmtech/KasmVNC/assets/5740797/8862ed70-5f1a-41bb-8d33-d67278476bf6)

**Reply by webysther** [reply]
Check kernel version, my Synology have a old kernel and some docker images stop working after a while.

**Comment by mmcclaskey** [comment]
Those logs from KasmVNC look normal in terms of KasmVNC is running and taking requests. I did notice the log that your IP was blacklisted. Which means you are sending the wrong credentials and KasmVNC blocked you after too many requests with bad creds.

**Reply by alhelo** [reply]
Thanks for reply.. What are the credentials ?
And how can I remove the Blacklist?
